crates/bevy_internal/src/lib.rs

```diff
@@ -1,143 +1,71 @@
-/// `use bevy::prelude::*;` to import common components, bundles, and plugins.
-pub mod prelude;
-
 mod default_plugins;
+#[doc(hidden)]
```

Comment:
Does `#![doc(hidden)]` at the top of the file work? And should it even be hidden at all if most users likely won't look at the docs of this crate?

Reply:
I'm not sure, maybe it not being hidden would be fine? The `#![doc(hidden)]` at the top of the file would also hide `MinimalPlugins` and `DefaultPlugins`